inbox/contacts/icloud.py

```diff
@@ -13,7 +13,7 @@
 from inbox.models.backends.generic import GenericAccount
 
 
-ICLOUD_CONTACTS_URL = 'https://contacts.icloud.com/'
+ICLOUD_CONTACTS_URL = 'https://contacts.icloud.com'
```

Comment:
Isn't the problem that we're not joining URLs correctly below (e.g: line 79, 84 & al)?